### ISSUE TITLE
loss is always 12 up and down

### ISSUE THREAD

**Issue by zhengshangdong** [meijieru/crnn.pytorch#193]
@meijieru 
Hi sir，I train with datasets——Synth90k .I use pytorch1.0 and python3.6.When I run the
 create_datasets.py，It tell me that I must encode my datasets into bytes.I finish it.But,when I run the train.py,the loss is always 12 up and down.I tried to divide train sets about 7,000,000 and 800,000 and 5000 pictures.These loss is not change.Do you kow why....Thanks

**Comment by zhengshangdong** [comment]
sorry,sir.It's my fault.I get the false labels....I thought the dataset——Synth90k is all number picture,so I wrote ： index = imagePath.find('.jpg') label = imagePath[index:index-4].It should be: imagePath.split('_')[-2].Thank you for your time.
Other men or women,I use pytorch 1.0 and cuda 10 and python3.6,this project is running.

**Comment by wutianye** [comment]
Hi，can you tell me what "It tell me that I must encode my datasets into bytes" means.
I change the code 
1.  with open(imagePath, 'rb') as f:       
2.  txn.put(k.encode(), v.encode())

but the dataset.py have  the log:
convert Unicode to bytes; use .encode()

**Comment by wutianye** [comment]
> sorry,sir.It's my fault.I get the false labels....I thought the dataset——Synth90k is all number picture,so I wrote ： index = imagePath.find('.jpg') label = imagePath[index:index-4].It should be: imagePath.split('_')[-2].Thank you for your time.
> Other men or women,I use pytorch 1.0 and cuda 10 and python3.6,this project is running.

Hi，can you tell me what "It tell me that I must encode my datasets into bytes" means.
I change the code 
1.  with open(imagePath, 'rb') as f:       
2.  txn.put(k.encode(), v.encode())

but the dataset.py have  the log:
convert Unicode to bytes; use .encode()
